intellij-idea/.idea/runConfigurations/Run_SQL_Migrations_.xml

```diff
@@ -2,7 +2,7 @@
   <configuration default="false" name="Run SQL Migrations " type="Application" factoryName="Application">
     <option name="MAIN_CLASS_NAME" value="de.metas.migration.cli.workspace_migrate.Main" />
     <module name="de.metas.migration.cli" />
-    <option name="VM_PARAMETERS" value="-Ddb.url=jdbc:postgresql://localhost:30040/metasfresh" />
+    <option name="VM_PARAMETERS" value="-Ddb.url=jdbc:postgresql://localhost:5433/metasfresh" />
```

Comment:
same remark about changing ports

Reply:
here it's a different beast. the standard port is 5432, and i shall change to that port (->TODO me)